custom-checks/checkstyle/src/main/java/org/openhab/tools/analysis/checkstyle/NoEmptyLineSeparatorCheck.java

```diff
@@ -62,11 +62,17 @@
  * </ul>
  *
  * @author Svilen Valkanov - Initial contribution
+ * @author Tanya Georgieva - Added check for switch statement without braces
  */
 public class NoEmptyLineSeparatorCheck extends AbstractCheck {
 
     private static final String MSG_LINE_AFTER_OPENING_BRACE_EMPTY = "Remove empty line after opening brace";
     private static final String MSG_LINE_BEFORE_CLOSING_BRACE_EMPTY = "Remove empty line before closing brace";
+    private static final String MSG_LINE_FOR_CASE_WITHOUT_BRACES_EMPTY = "Remove empty line";
```

Comment:
You have a really generic message maybe you should give the constant a generic name as well?

Reply:
Yes, I agree. I will change it.